src/components/Staking/StakingStatsBox.tsx

```diff
@@ -177,7 +170,16 @@ const StakingStatsBox: React.FC<IProps> = () => {
         )}
         <Label>
           <Translation id="page-staking-stats-box-metric-1" />
-          <BeaconchainTooltip />
+          <BeaconchainTooltip>
+            <div>
```

Comment:
Not blocking but we could probably lose this div now?

Reply:
Will double check that `<BeaconchainTooltip>` doesn't require a single child

Reply:
...it does not, will patch, thanks!